README.md

```diff
@@ -1 +1,37 @@
-# mobile
\ No newline at end of file
+# My Portfolio
+
+This is a practice project about building a mobile and website version of my portfolio.
+
+
+## Built With
+
+- Major languages : Html, Css, JavaScript
+- No Frameworks
+
+## Authors
+
+👤 **Amine**
+
+- GitHub: [@githubhandle](https://github.com/medaminedev66)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-amine-smahi-1b8615187)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] I noticed your **MIT Licence** is not a link. Please, be sure to make it as a link to your MIT LICENSE file. Having a broken link in your README.md file is unprofessional I think.